Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -2,6 +2,7 @@ import Foundation
 @_spi(Restricted) import MapboxMaps
 import MapboxCoreNavigation
 import MapboxDirections
+import os.log
```

Comment:
It'd be useful to have logging information when `CarPlayActivity` value changes.